extension/src/popup/basics/Images.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+interface RetinaImgProps {
+  retina: string;
+  src?: string;
+  alt: string;
+}
+
+export const RetinaImg = ({ retina = "", src = "", alt }: RetinaImgProps) => (
+  <img src={src || retina} srcSet={`${retina} 2x`} alt={alt} />
```

Comment:
Why offer the choice? Convention is usually better over configuration.

Reply:
Hm. Not sure I understand. Are you asking why include `src` as a prop?

Reply:
I read this as, `src` + `retina` are already follow a `x@2x` naming convention, so it'd be simpler to assume the `@2x` exists and skip the `retina` prop altogether